tock/hours/models.py

```diff
@@ -265,6 +269,8 @@ def save(self, *args, **kwargs):
             self.billable_expectation = self.user.user_data.billable_expectation
         if self.id:
             self.calculate_hours()
+            self.organization = self.user.user_data.organization
+            self.unit = self.user.user_data.unit
```

Comment:
We should set these when a timecard is first created to avoid overwriting these fields if/when a timecard is updated at a later time.